### ISSUE TITLE
[CCE][NODE] Assigned EIP is not deletable using TF

### ISSUE THREAD

**Issue by Moep90** [opentelekomcloud/terraform-provider-opentelekomcloud#678]
# Terraform Version
```
$ terraform --version
Terraform v0.13.5
+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.19.5
```

# Affected Resource(s)
- opentelekomcloud_cce_node_v3.cce_node

# Terraform Configuration Files
```hcl
resource "opentelekomcloud_cce_node_v3" "cce_node" {

  count      = var.cce_nodes
  cluster_id = var.cce_cluster_id

  flavor_id = var.cce_node_flavor
  key_pair  = var.key_pair_name

  iptype         = null
  bandwidth_size = null
  sharetype      = null
  eip_ids        = null
  eip_count      = null

  availability_zone = element(var.availability_zones, count.index+1)

  root_volume {
    size       = var.cce_node_root_volume_size
    volumetype = var.cce_node_volume_type
  }
  data_volumes {
    size       = var.cce_node_data_volume_size
    volumetype = var.cce_node_volume_type
  }

  timeouts {
    create = "30m"
    delete = "30m"
  }
}
```
# Steps to Reproduce
1. `terraform apply`

# Expected Behavior
- Existing nodes should loose their EIP (without recreation)
- New nodes shouldn't be created with EIP
# Actual Behavior
- Existing nodes are ignored regardless of the setting (see example)
- New nodes are created without an EIP

# References
- #533

**Comment by Moep90** [comment]
UPDATE:
```
$ terraform --version
Terraform v0.13.5
+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.21.1
```

# Actual Behavior
* Existing nodes will be recreated 
```
      + bandwidth_charge_mode      = (known after apply)
      - bandwidth_size             = 100 -> null # forces replacement
```

**Comment by outcatcher** [comment]
Postponed to v1.21.5 due to CCE testing problems

**Comment by outcatcher** [comment]
Implemented in release v1.21.5
